snuba/consumers/consumer_config.py

```diff
@@ -170,6 +175,7 @@ def resolve_consumer_config(
         max_batch_size=max_batch_size,
         max_batch_time_ms=max_batch_time_ms,
         env=resolved_env_config,
+        kafka_consumer_config=kafka_consumer_config,
```

Comment:
Please don't add a new item here, Kafka consumer config should be part of the existing`raw_topic` configuration item